manifest.tmpl

```diff
@@ -0,0 +1,17 @@
+image: rancher/fleet:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-head{{/if}}
+manifests:
+  -
+    image: rancher/fleet:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-head{{/if}}-amd64
```

Comment:
Since your image format will look like `$tag-linux-$arch`, shouldn't the image in manifest follow the same format?

Reply:
Good point; changing this to match the format.